library/std/src/collections/hash/map.rs

```diff
@@ -200,6 +200,8 @@ use crate::sys;
 
 #[cfg_attr(not(test), rustc_diagnostic_item = "hashmap_type")]
 #[stable(feature = "rust1", since = "1.0.0")]
+#[doc(alias = "dictionary")]
+#[doc(alias = "Map")]
```

Comment:
Isn't it called `HashMap` in Java? Also, you might want to add `dict` instead/in addition.

Reply:
Map is the trait, HashMap is the instance. `dict` is a good idea, thanks :)

Reply:
The problem with having `Map` go here for Java is that there are other Rust types that would implement `Map` if they were in Java — e.g., `BTreeMap` — so this is not the "one" place we would want people to go when searching (unless I'm misunderstanding how doc aliases work, which I might be).

Reply:
Sure, I can also add "map" to BTreeMap if you like.

Reply:
I was suggesting that you just remove the `Map` doc alias altogether. But if you think it makes sense, I won't get in the way :)

Reply:
Ok, I added the alias to BTreeMap too.